frontend/components/ClubEditPage/EventsCard.tsx

```diff
@@ -292,7 +292,22 @@ const eventTableFields = [
   {
     name: 'is_ics_event',
     label: 'ICS',
-    converter: (a: boolean): ReactElement => <Icon name={a ? 'check' : 'x'} />,
+    converter: (a: boolean): ReactElement => (
+      <Icon
+        className={`has-text-${a ? 'has-text-success' : 'has-text-danger'}`}
```

Comment:
Not sure if a class name with `has-text-has-text-success` or `has-text-has-text-danger` exists. Could you check if this is a typo?